### PR TITLE
add footer and header to didit login

### PR DESCRIPTION
**Pull request by Keddib**
add footer and header as react node props on login component for more customization 

Screenshot 2023-11-14 at 14 47 55
Screenshot 2023-11-14 at 14 48 23

### CODE REVIEW

File: examples/with-nextjs/src/pages/index.tsx [hunk 1]
```diff
@@ -134,12 +134,52 @@ export default function Home() {
...
-                  <DiditLogin mode={DiditLoginMode.EMBEDDED} />
+                  <DiditLogin
+                    mode={DiditLoginMode.EMBEDDED}
+                    header={
+                      <div className="w-full">
+                        <h1 className="m-0 mb-4 text-2xl font-medium">
+                          Signup to
+                          <strong className="text-blue-700"> Didit</strong>
+                        </h1>
+                        <p>All you issentials in one place</p>
```
Comment (arosasg) [inline examples/with-nextjs/src/pages/index.tsx:151]: All your life essentials in one place

File: examples/with-nextjs/src/pages/index.tsx [hunk 2]
```diff
@@ -132,14 +132,54 @@ export default function Home() {
...
+                        </a>
+                      </div>
+                    }
+                    header={
+                      <div className="w-full">
+                        <h1 className="m-0 mb-4 text-2xl font-medium">
+                          Signup to
+                          <strong className="text-blue-700"> Didit</strong>
+                        </h1>
+                        <p>All you issentials in one place</p>
```
Comment (pablogamium) [inline examples/with-nextjs/src/pages/index.tsx:151]:
```suggestion
                        <p>All your essentials in one place</p>
```
?

**Review by arosasg — APPROVED** [review]

**Review by pablogamium — APPROVED** [review]
LGTM

File: examples/with-nextjs/src/pages/index.tsx (hunk 1 above)
Comment (Keddib) [reply examples/with-nextjs/src/pages/index.tsx]: fixed

**Pull request merged**